stories/multiple-users.md

```diff
@@ -0,0 +1,46 @@
+# Feature - Multiple users per app
+
+The team works a sustainable pace, leaving their computers at work and pairing
+even. That has meant that people are often switching computers. Although this
+hasn't been an issue for the application creator, who only solos, the rest of
+the team wants to see their own tasks, and not left overs. So, we need to allow
+user login and have separate data per user login.
+
+Happily a password is not required for this feature, but given this troublesome
+crowd, it may be coming soon.
```

Comment:
Is this "coming soon" flavor text, or is there a missing feature about passwords?

Reply:
It's just a hint to not optimize in advance more than anything. I don't think there should be a feature for password.